tests/suites/test_suite_psa_crypto.function

```diff
@@ -2262,6 +2286,12 @@ void mac_sign( int key_type_arg,
                                      actual_mac, mac_buffer_size,
                                      &mac_length ) );
 
+    /* Ensure double sign fails properly. */
+    TEST_EQUAL( psa_mac_sign_finish( &operation,
```

Comment:
Please avoid mixing input validation tests with operation correctness tests. That makes it harder to understand what is tested, and makes it harder to run just the input validation tests without also running the more costly operation correctness tests. This function is for operation correctness.

This also goes for `mac_verify`.

Also, why test `sign_finish` followed by `sign_finish`, but not other invalid combinations such as `sign_finish` followed by `update`, `sign_setup` followed by `verify_finish`, etc.?